benches/bench.rs

```diff
@@ -5,7 +5,7 @@ extern crate test;
 
 use std::iter;
 
-use aho_corasick::{Automaton, AcAutomaton, Transitions};
+use aho_corasick::{AcAutomaton, Automaton, Transitions};
```

Comment:
Sorry, but please remove the commit that runs rustfmt. If there isn't a `rustfmt.toml` in a repo, then it's probably best to assume that it isn't being used.

I realize that this is probably going to be a pain to unwind this, but rustfmt is not a place where I'm ready to use it yet.